src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -779,8 +779,11 @@ private synchronized Set<String> getExecuteWorkers() throws IOException {
       return recentExecuteWorkers.get();
     } catch (RuntimeException e) {
       // unwrap checked exception mask
+      log.log(Level.WARNING, "getExecuteWorkers failed", e);
       Throwable cause = e.getCause();
-      Throwables.throwIfInstanceOf(cause, IOException.class);
+      if (cause != null) {
+        Throwables.throwIfInstanceOf(cause, IOException.class);
+      }
       throw e;
```

Comment:
Prefer a wrap in IOException in this case - I want to fit within the expectation of IOException, and to figure out how it's possible that this happened.